tests/test_framework/mod.rs

```diff
@@ -589,6 +608,10 @@ impl TestFramework {
             generate_blocks(&tf_clone.bitcoind, 10);
         });
 
+        for taker in &takers {
+            let _ = taker.delete_offer_cache();
+        }
+
```

Comment:
This should not be required as each test runs in a dedicated data directory. So, the offer data should be empty at the start of each test.